### PR TITLE
Don't require password field for PUT /users/<id>

### PR DESCRIPTION
**Pull request by justinvdm**
At the moment, we get the following response:

```http
HTTP/1.0 400 Bad Request
{"password":["This field is required."]}
```

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler ready for review

File: authapi/serializers.py [hunk 1]
```diff
@@ -105,3 +97,25 @@ def update(self, instance, validated_data):
...
+
+class NewUserSerializer(BaseUserSerializer):
+    password = serializers.CharField(
+        style={'input_type': 'password'}, write_only=True)
+
+    class Meta:
+        model = User
+        fields = (
+            'id', 'url', 'first_name', 'last_name', 'email', 'admin', 'teams',
+            'organizations', 'password', 'active')
```
Comment (rudigiesler) [inline authapi/serializers.py]: We can probably have the `Meta` on the base class, and just adjust the CharField on these classes. Is there a reason why you chose this way?

**Comment by rudigiesler** [comment]
Just the one comment, looks good otherwise.

File: authapi/serializers.py (hunk 1 above)
Comment (justinvdm) [inline authapi/serializers.py]: Ah, didn't think of just adjusting the field, sounds good.

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**